python/setup.py

```diff
@@ -71,19 +71,23 @@
 optional_ray_files = ["ray/nightly-wheels.yaml"]
 
 ray_autoscaler_files = [
-    "ray/autoscaler/aws/defaults.yaml", "ray/autoscaler/azure/defaults.yaml",
-    "ray/autoscaler/azure/azure-vm-template.json",
-    "ray/autoscaler/azure/azure-config-template.json",
-    "ray/autoscaler/gcp/defaults.yaml", "ray/autoscaler/local/defaults.yaml",
+    "ray/autoscaler/aws/defaults.yaml",
+    "ray/autoscaler/azure/defaults.yaml",
+    "ray/autoscaler/_private/azure/azure-vm-template.json",
```

Comment:
why are those in _private?

Reply:
There was some refactoring of the code and node provider specific python files and arm templates we're moved, but weren't updated here.